dashboard/app/models/user.rb

```diff
@@ -114,6 +114,9 @@ class User < ActiveRecord::Base
     last_seen_school_info_interstitial
     ui_tip_dismissed_homepage_header
     ui_tip_dismissed_teacher_courses
+    oauth_refresh_token
+    oauth_token
+    oauth_token_expiration
```

Comment:
Though I presume you are aware of it, I want to note the `remember_created_at` column on `users` that we use for this purpose for password sign-ins. No action expected.